force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -237,12 +323,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return A Set of unique strings.
      */
     public static Set<String> createStringSet(List<String> inputList) {
+        
         // Create a new Set of strings
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
+        for (String dupe : inputList) {
+            stringSet.add(dupe);
+        }
```

Comment:
There is a simpler way to do this bu just using the Set<> Constructor to pass in a list. That way you dont need the loop at all.

Reply:
Ah sweet i see that now, thanks!
https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_set.htm#apex_System_Set_ctor